website/src/redux/actions/ElementActions.jsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * @file File of all the possible actions we can take on the Redux store.
+ */
+
+import {
+  setElementLocation, setElementSize, setElementRotation
+} from '../../firebase-utils';
+import {
+  INIT_ELEMENTS, UPDATE_POSITION, UPDATE_SIZE, UPDATE_ROTATION,
+  TARGET_ELEMENT
+} from './ActionConstants';
+
+
+/**
+ * Gives the action for initializing positions.
+ * @param {Object} elemList Object of the element details.
+ * @returns {Object} The object detailing the action details.
+ */
+export const initElements = elemList => ({
+  type: INIT_ELEMENTS,
+  elements: elemList,
+});
+
+/**
+ * Gives the action for updating a position.
+ * @param {String} elementId String of the elment ID.
+ * @param {Object} updatedPosition Object detailing the updated location.
+ * @returns {Object} The object detailing the action details.
+ */
+export const updatePosition = (elementId, updatedPosition) => ({
```

Comment:
Good point, I changed it just to one function and takes a type argument. Now updateAndPersist takes a type instead of a specific action too